org.code.toboggan.network/META-INF/MANIFEST.MF

```diff
@@ -12,3 +12,4 @@ Require-Bundle: org.eclipse.core.runtime,
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-ActivationPolicy: lazy
 Export-Package: org.code.toboggan.network
+Import-Package: com.google.common.collect
```

Comment:
Where does this package come from? As a general pattern, we are importing plugin bundles instead of individual packages. If this is exported by our clientcore, then add clientcore as a plugin dependency instead of just the package specification.